docs/unified-test-documentation/dasharo-compatibility/328-logo-customization-functionality.md

```diff
@@ -0,0 +1,125 @@
+# Dasharo Compatibility: Logo customization functionality
+
+## Test cases common documentation
+
+**Test setup**
+
+1. Proceed with the
+    [Generic test setup: firmware](../../generic-test-setup/#firmware).
+1. Proceed with the
+    [Generic test setup: OS installer](../../generic-test-setup/#os-installer).
+1. Proceed with the
+    [Generic test setup: OS installation](../../generic-test-setup/#os-installation).
+1. Proceed with the
+    [Generic test setup: OS boot from disk](../../generic-test-setup/#os-boot-from-disk).
+
+## LCM001.001 Replace logo in existing image and flashing firmware
+
+**Test description**
+
+This test aims to verify whether it's possible to replace logo in existing
```

Comment:
Is the purpose of this test to flash the device with the firmware? In my opinion, the primary goal is to check if the correct logo appears on the boot screen after flashing.

Reply:
changed description